source/declarations.tex

```diff
@@ -2987,11 +2988,8 @@
 \end{example}
 
 \pnum
-A pointer to member shall not point to a static member
-of a class\iref{class.static},
-a member with reference type,
-or
-``\cv{}~\tcode{void}''.
+A pointer to member shall not designate a static member
+of a class\iref{class.static}.
```

Comment:
This restriction seems redundant; how could the "static member" situation ever arise?
&C::static_member yields a regular pointer, not a pointer to member.

Reply:
True; I didn't want to drastically change it _too_ much, be this can certainly be turned into a note saying something along the lines of: "A pointer to member designating a static member cannot be formed as there is no syntax to do so."